core/src/main/java/com/linecorp/armeria/server/ServiceConfigBuilder.java

```diff
@@ -44,47 +47,47 @@
         this.service = requireNonNull(service, "service");
     }
 
-    @Nullable
-    Long requestTimeoutMillis() {
-        return requestTimeoutMillis;
+    @Override
+    public ServiceConfigSetters requestTimeout(Duration requestTimeout) {
```

Comment:
You sometimes return `ServiceConfigSetters` and sometimes `ServiceConfigBuilder` in this class. It doesn't hurt to return `ServiceConfigSetters` because it's package-private, but perhaps it's better returning `ServiceConfigBuilder` for consistency.

Reply:
Oops.. Thanks for pointing!